### PR TITLE
fix empty docs

### PR DESCRIPTION
**Pull request by buzzdecafe**

### CODE REVIEW

File: src/empty.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ var _hasMethod = require('./internal/_hasMethod');
 
 
 /**
- * `empty` wraps any object in an array. This implementation is compatible with the
+ * `empty` returns an empty list for any argument. This implementation is compatible with the
```
Comment (davidchambers) [inline src/empty.js]: I would say _`empty` returns the empty value of its argument's type_, since the empty value needn't be a list.

Comment (buzzdecafe) [inline src/empty.js]: that's definitely better, but not quite right I think. If you call `R.empty("abc")`you get back `[ ]`

Comment (buzzdecafe) [inline src/empty.js]: looking over the docs, we are documenting the list case, and leaving the dispatch case as a delightful easter egg for users to discover.

File: src/empty.js [hunk 2]
```diff
@@ -3,8 +3,8 @@ var _hasMethod = require('./internal/_hasMethod');
 
 
 /**
- * `empty` wraps any object in an array. This implementation is compatible with the
- * Fantasy-land Monoid spec, and will work with types that implement that spec.
+ * `empty` returns an empty list for any argument, except when  the argument satisfies the
+ * Fantasy-land Monoid spec. In that case, this function will invoke `empty` on that Monoid.
```
Comment (davidchambers) [inline src/empty.js]:
```diff
-In that case, this function will invoke `empty` on that Monoid.
+In that case, this function will return the result of invoking `empty` on that Monoid.
```

Comment (buzzdecafe) [inline src/empty.js]: yep. i think that's it.

File: src/empty.js [hunk 3]
```diff
@@ -3,8 +3,9 @@ var _hasMethod = require('./internal/_hasMethod');
 
 
 /**
- * `empty` wraps any object in an array. This implementation is compatible with the
- * Fantasy-land Monoid spec, and will work with types that implement that spec.
+ * `empty` returns an empty list for any argument, except when  the argument satisfies the
```
Comment (davidchambers) [inline src/empty.js]: Nit: You have two spaces between "when" and "the".

Comment (buzzdecafe) [inline src/empty.js]: your copy editing skills continue to astound

**Comment by davidchambers** [comment]
:deciduous_tree:

**Comment by CrossEye** [comment]
Definitely an improvement.  But is it oversold?  `empty` will dispatch to the object if it has any `empty` function, regardless if that object implements `Monoid`, correct?

**Comment by buzzdecafe** [comment]
that's why i used "satisifies" instead of "implements"

**Pull request merged**